ResourceManager/GitExtensionsFormBase.cs

```diff
@@ -56,6 +56,11 @@ public virtual bool ProcessHotkey(Keys keyData)
             return hotkey is not null && ExecuteCommand(hotkey.CommandCode);
         }
 
+        internal bool DispatchCommand(int command)
+        {
+            return ExecuteCommand(command);
+        }
+
```

Comment:
not needed then

Reply:
Would be anyway, because, AFAIK, a method cannot be simultaneously `internal` and `protected`. And it probably should not be `public`.

Reply:
> Would be anyway, because, AFAIK, a method cannot be simultaneously internal and protected.

Indeed. This is a strange restriction IMO.

>  And it probably should not be public.

`ProcessCmdKey` is public, too, i.e. should not be a problem.

Reply:
What I mean is to declare `ExecuteCommand` public.

Reply:
It's a moot point anyway, since bubbling cannot be done inside `ExecuteCommand` (it would conflict with the OS bubbling), so `GitExtensionsControl` would necessarily have `DispatchCommand`. Uniformity of API demands `GitExtensionsFormBase` has it too.

Reply:
Please do not tell me the following was consistent:
```
class GitExtensionsControl
{
        public bool DispatchCommand(int command)
        {
            for (Control control = this; control != null; control = control.Parent)
            {
                if (control is GitExtensionsControl extensionsControl && extensionsControl.ExecuteCommand(command))
                {
                    return true;
                }

                if (control is GitExtensionsFormBase extensionsForm && extensionsForm.DispatchCommand(command))
                {
                    return true;
                }
            }

            return false;
        }
}

public class GitExtensionsFormBase
{
        internal bool DispatchCommand(int command)
        {
            return ExecuteCommand(command);
        }
}
```

You can keep `GitExtensionsControl.DispatchCommand`, but there is no reason to have it in `GitExtensionsFormBase`.

Reply:
Accepted